src/components/Comment/styles.module.scss

```diff
@@ -0,0 +1,26 @@
+@import '../../scss/variables.scss';
+
+.container {
+  padding: $default-padding;
+  width: 100%;
+}
+
+.comment,
+.form {
+  background-color: $light-gray;
+  border: 1px solid darken($light-gray, 10);
+  padding: $default-padding;
+}
+
+.textarea {
+  display: block;
+  margin-bottom: $default-padding;
+  width: 100%;
+}
+
+.comment {
+  display: grid;
```

Comment:
The defaults for flex seem to accomplish the same layout:

```diff
diff --git a/src/components/Comment/styles.module.scss b/src/components/Comment/styles.module.scss
index 42d5a91..04ac44f 100644
--- a/src/components/Comment/styles.module.scss
+++ b/src/components/Comment/styles.module.scss
@@ -19,8 +19,6 @@
 }
 
 .comment {
-  display: grid;
-  // The FontAwesome icon size is 32px.
-  grid-template-columns: 32px + $default-padding auto;
+  display: flex;
   width: 100%;
 }
```

Am I missing something? I checked the storybook and I think this does what we want.

Reply:
ah, right..